src/leb.rs

```diff
@@ -86,26 +90,28 @@ mod tests {
         assert_eq!(buf[..i], [0x80, 1]);
         buf.iter_mut().for_each(|b| *b = 0x55);
 
-        let i = leb64((1 << 32) - 1, &mut buf);
-        assert_eq!(buf[..i], [0xff, 0xff, 0xff, 0xff, 0xf]);
-        buf.iter_mut().for_each(|b| *b = 0x55);
+        if cfg!(target_pointer_width = "64") {
```

Comment:
another nitpick: it's not very intuitive to figure out what's being tested here... Personally I'd put each check in a dedicated test case anyway (this way they're independent and their purpose is documented on the fly) but in this constellation comments would be helpful and time-saving